azdev/operations/constant.py

```diff
@@ -0,0 +1,158 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
```

Comment:
Better to put this file in azure-cli repo; otherwise we need release a new version for azdev if any update in this file.

Reply:
May I ask where should I put this file ？
Can I put this file in here? 
`azure-cli\src\azure-cli\cmdcov_config.py`